packages/manager/modules/advices/src/advices/component.js

```diff
@@ -0,0 +1,71 @@
+import controller from './controller';
+import template from './template.html';
+
+/**
+ * Makes 2api call to /adives to get advices for a given service type and name.
+ * It used serviceType, serviceName and queryParams to construct the URl.
+ *
+ * serviceType: type of service, refer 2api /advices documentation for list of service types supported.
```

Comment:
Is the /advices schema exposed ? If not it could be hard to refer to this documentation

Reply:
No, it's not, the service name is given while implementing the 2api in the route file. The same service name has to be used in API to match it. I have changed the description.